lib/tasks/offers_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
I think this is causing staging deploys [to fail recently](https://app.circleci.com/pipelines/github/artsy/convection?branch=master). It should go under `spec/` if it's a test, so the deployed environments don't try to load it.